sdk/core/core-client/test/serializer.spec.ts

```diff
@@ -11,6 +11,7 @@ import {
   CompositeMapper,
 } from "../src";
 import { Mappers } from "./testMappers";
```

Comment:
nit: should `testMappers.ts` be renamed to `testMappers1.ts`?

Reply:
done